sdk/src/main/scala/com/horizen/account/storage/AccountStateMetadataStorageView.scala

```diff
@@ -214,17 +208,17 @@ class AccountStateMetadataStorageView(storage: Storage) extends AccountStateMeta
         WithdrawalEpochCertificateSerializer.toBytes(certificate)))
     })
 
-    withdrawalEpochInfoOpt.foreach(epochInfo => {
-      // Update BlockFeeInfo data
+    blockFeeInfoOpt.foreach(feeInfo => {
+      val epochInfo = withdrawalEpochInfoOpt.getOrElse(getWithdrawalEpochInfo)
```

Comment:
can be just `val epochInfo = getWithdrawalEpochInfo`
but maybe this way is more clear